Files/Controllers/TerminalController.cs

```diff
@@ -70,8 +70,8 @@ private async Task Load()
 
         public async void Init()
         {
-            await Load();
-            await GetInstalledTerminals();
+            //await Load();
+            //await GetInstalledTerminals();
```

Comment:
I've commented these out for the time being because the Load(); was crashing the app for me. I'll uncomment these once the PR is done